lesson03/assignment/index.js

```diff
@@ -0,0 +1,34 @@
+//Create object with data from JSON file
+const guys = [
+    { name: "Steve Smith", jobTitle: "Project Manager", company: "Front End Dev Co", experience: "3 years", school: "UW", major: "Marketing", email: "steve@fedc.com", linkedInUrl: "steve.linkedinprofile.com" },
+    { name: "Aaron Katz", jobTitle: "Full Stack Developer", company: "Web Sites and More", experience: "5 years", school: "SU", major: "Computer Science", email: "aaronNotMyemail@uw.com", linkedInUrl: "aaron.linkedinprofile.com"},
+    { name: "Kyle Hendricks", jobTitle: "Starting Pitcher", company: "Chicago Cubs", experience: "12 years", school: "USC", major: "Pitching", email: "kyleH@cubs.com", linkedInUrl: "kyle.linkedinprofile.com"},
+    { name: "Michael Jordan", jobTitle: "Point Guard", company: "Chicago Bulls", experience: "20 years", school: "UNC", major: "Trash Talking", email: "mJordan@bulls.com", linkedInUrl: "mJordan.linkedinprofile.com"},
+];
+
+//Create variable containing html and map through elements of object above
+const myCode = `
+    ${guys.map(guys => `
```

Comment:
Convention is that the temporary variable is singular, when the variable for the whole collection is plural. The temporary variable should be different, and making it singular makes the code inside the block easier to read. In this case, it might look like "guy" vs "guys": 
```
${guys.map(guy => `
    <h1 class="photo-title__name">${guy.name}</h1>
`
)}
```